### PR TITLE
added some NM drops. removed Agar Agar's skill list.

### PR DESCRIPTION
**Pull request by wrenffxi**
added or fixed drops for several NMs
removed MP from non-caster NMs: Hawkeyed Dnatbat, Firedance Magmaal Ja, Sugaar, Ratatoskr
increased level range min for NM: Ratatoskr
removed skill list from NM: Agar Agar (does not use TP moves)

### CODE REVIEW

File: sql/mob_droplist.sql [hunk 1]
```diff
@@ -32387,7 +32387,7 @@ INSERT INTO `mob_droplist` VALUES (4440,0,0,1000,1039,70);
 INSERT INTO `mob_droplist` VALUES (4441,0,0,1000,2159,420);
 INSERT INTO `mob_droplist` VALUES (4441,0,0,1000,2165,30);
 INSERT INTO `mob_droplist` VALUES (4441,0,0,1000,2220,220);
-INSERT INTO `mob_droplist` VALUES (4442,0,0,1000,17709,800);
+INSERT INTO `mob_droplist` VALUES (4442,0,0,1000,17709,583); (Cwn Cyrff) Swan Bilbo
```
Comment (TeoTwawki) [inline sql/mob_droplist.sql]: Importing mob_droplist.sql into the database...ERROR 1064 (42000) at line 32390: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'Cwn Cyrff) Swan Bilbo
INSERT INTO `mob_droplist` VALUES (4443,0,0,1000,858,290)' at line 1

**Comment by TeoTwawki** [comment]
gdi github, stop being stupid

File: sql/mob_droplist.sql (hunk 1 above)
Comment (TeoTwawki) [inline sql/mob_droplist.sql]: Importing mob_droplist.sql into the database...ERROR 1064 (42000) at line 32390: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'Cwn Cyrff) Swan Bilbo
INSERT INTO `mob_droplist` VALUES (4443,0,0,1000,858,290)' at line 1

missed the `-- `

Comment (wrenffxi) [reply sql/mob_droplist.sql]: fixed

**Pull request merged**